include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -5994,7 +5994,7 @@ namespace VectorTools
                       AssertIndexRange(associated_face_dof_index,
                                        associated_face_dof_to_face_dof.size());
                       associated_face_dof_to_face_dof
-                        [associated_face_dof_index] = quad_dof_idx;
+                        [associated_face_dof_index] = face_idx;
```

Comment:
I am a bit confused by the naming of the variable on the left hand side then. I read that as mapping from dof to dof, but you assign to it a face index. That seems confusing ... ... but then I scroll up, and `face_idx` is actually a DoF index after all. So yes, I think that might be ok.